f/groupMarkers/f_setLocalGroupMarkers.sqf

```diff
@@ -23,12 +23,12 @@ params [
 private _groups = [];
 
 // Markers seen by players in NATO, NATO (Pacific) & CTRG slots.
-if (_unitfaction in ["blu_f","blu_t_f","blu_ctrg_f","blu_w_f"]) then {
+if (_unitfaction in ["blu_f","blu_t_f","blu_ctrg_f","blu_w_f","nato","natowoodland","natopacific"]) then {
 	_groups = f_var_groupData_blufor_nato;
 };
 
-// Markers seen by players in FIA & CTRG slots.
-if (_unitfaction in ["blu_g_f","blu_ctrg_f"]) then {
+// Markers seen by players in FIA & CTRG slots. Add "fia" if using "fia" with Virtual Faction.
```

Comment:
Maybe worth having "fia" in the BLUFOR case by default (since that's what's in the template even though it doesn't use the "fia" faction alias) and have the comment be to move it into "opt_g_f" or "ind_g_f" if using those factions. Just a bit wary about "fia" not being in any case by default.